src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php

```diff
@@ -57,13 +56,12 @@
                 }
 
                 if (ACTIVE_TRANSLATION) {
-                    $bIsTextField = false;
-                    $aAllowedTypes = array('CMSFIELD_STRING', 'CMSFIELD_TEXT', 'CMSFIELD_STRING_UNIQUE');
+                    $lastFieldIsTextField = false;
```

Comment:
This variable means "is one line text field"?

And they are not "allowed types" but "special feature types"?